huntsman/scheduler/observation.py

```diff
@@ -113,3 +113,73 @@ def __init__(self, position, pattern=dither.dice9, n_positions=9,
 
         # Specify directory root for file storage
         self._directory = os.path.join(self._image_dir, 'flats')
+
+
+class DarkObservation(Observation):
+
+    """ Collection of darks when weather is bad
+
+    Dark observations will consist of both multiple exposure, but no fields.
+    Also, the mount will be parked when used.
+
+    Note:
+        For now the new observation must be created like a normal `Observation`,
+        with one `exptime` and one `field`. Then use direct property assignment
+        for the list of `exptime` and `field`. New `field`/`exptime` combos can
+        more conveniently be set with `add_field`
+    """
+
+    def __init__(self, position, *args, **kwargs):
+        """
+        Args:
+            position (str): Center of field, can be anything accepted by
+                `~astropy.coordinates.SkyCoord`.
+        """
+        # Create the observation
+        dark_coords = position.to_string('hmsdms')
+        dark_field = Field('Dark Field', dark_coords)
+        super().__init__(field=dark_field, *args, **kwargs)
+
+        # Set initial list to original values
+        self._exptime = listify(self.exptime)
+        self._field = listify(self.field)
+
+        self.extra_config = kwargs
+
+    @property
+    def exptime(self):
+        exptime = self._exptime[self.exposure_index]
+
+        if not isinstance(exptime, u.Quantity):
+            exptime *= u.second
+
+        return exptime
+
+    @exptime.setter
+    def exptime(self, values):
+        assert all(t > 0.0 for t in listify(values)), \
+            self.logger.error("Exposure times (exptime) must be greater than 0")
+
+        self._exptime = listify(values)
+
+    @property
+    def field(self):
+        return self._field[self.exposure_index]
+
+    @field.setter
+    def field(self, values):
+        assert all(isinstance(f, Field) for f in listify(values)), \
+            self.logger.error("All fields must be a valid Field instance")
+
+        self._field = listify(values)
+
+    @property
+    def exposure_index(self):
+        _exp_index = 0
+        with suppress(AttributeError):
+            _exp_index = self.current_exp_num % len(self._exptime)
+
+        return _exp_index
+
+    def __str__(self):
+        return "DarkObservation: {}: {}".format(self._field, self._exptime)
```

Comment:
Just a reminder you can get in the habit of:

```suggestion
        return f"DarkObservation: {self._field}: {self._exptime}"
```